Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_26.md

```diff
@@ -0,0 +1,4 @@
+
+#### Incident Fields
+
+Updated the **ASM - Protocol** fields to multiSelect type.
```

Comment:
Let's also add "Fixed an issue where..." for customers who already updated to the problematic version.

Reply:
@YuvHayun , updated [update RN](https://github.com/demisto/content/pull/32415/commits/07d37e7de80176e564a6e95000cb76de2c925bbd)